src/UIComponents/DataCard.jsx

```diff
@@ -24,6 +25,11 @@ class DataCard extends Component {
       );
     }
 
+    const removedRowsMsg =
+      removedRowsCount === 1
+      ? `${this.props.removedRowsCount} row was removed because it contained an empty cell.`
+      : `${this.props.removedRowsCount} rows were removed because they contained an empty cell.`;
```

Comment:
It seems slightly odd that we still generate this string for 0 removed rows, even though we don't show it later.